packages/docs/src/pages/components.js

```diff
@@ -5,17 +5,26 @@ import map from 'lodash.map';
 import slugify from 'slugify';
 import PropTypes from 'prop-types';
 import 'holderjs';
+import PrismCode from 'react-prism';
 
-const Component = ({ html, title }) => (
+require('prismjs');
+require('prismjs/themes/prism.css');
+
+const Component = ({ html, title, id }) => (
   <div>
-    <h3>{title}</h3>
+    <h3 id={id}>{title}</h3>
     <div className="docs-example"> {ReactHtmlParser(html)}</div>
+    <button className="btn btn-default mb-4"><a href={`#${id}-code`} data-toggle="collapse">View Code </a><i className="icon icon-code" /></button>
```

Comment:
A link inside a button? Why not just add `className="btn btn-default mb-4"` to the link to style it like a button?
As it is right now, clicking button which isn't the link (such as the padding and the code icon) will not toggle the code.

Reply:
Why not use `slugify`?